roles/matrix-base/templates/static-files/well-known/matrix-support.j2

```diff
@@ -0,0 +1,9 @@
+#jinja2: lstrip_blocks: "True"
+{
+	{% if matrix_homeserver_admin_contacts %}
+    "admins": {{ matrix_homeserver_admin_contacts }}
```

Comment:
If `matrix_homeserver_admin_contacts` is an actual list (as discussed in my other comment), you can do this:

```suggestion
	{% if matrix_homeserver_admin_contacts|length > 0 %}
    "admins": {{ matrix_homeserver_admin_contacts|to_json }}
```

Reply:
Resolved in [32a604d](https://github.com/spantaleev/matrix-docker-ansible-deploy/pull/1931/commits/32a604d7c16b28dad552c8c87be710bdc9e37d92)